sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClientJavaDocCodeSnippets.java

```diff
@@ -24,4 +43,868 @@ public TextAnalyticsAsyncClient createTextAnalyticsAsyncClient() {
         // END: com.azure.ai.textanalytics.TextAnalyticsAsyncClient.instantiation
         return textAnalyticsAsyncClient;
     }
+
+    // Languages
+
+    /**
+     * Code snippet for detecting Language
```

Comment:
```suggestion
     * Code snippet for {@link TextAnalyticsAsyncClient #detectLanguage(String)}
```

Reply:
And same for rest of the file

Reply:
This block stands for all overloads of detect language api

Reply:
we should keep each BEGIN and END for a separate method. So It is easier to maintain. You could refer to AzConfig or Storage